Global Shark Attack Dataset/README.md

```diff
@@ -0,0 +1,15 @@
+# Description
+
+Contains the data of location of past shark attack in the world
+
+# Details
+
+All of the data on this site comes from the Global Shark Attack File (GSAF), a spreadsheet of human/shark interactions, compiled by the Shark Research Institute. It is hoped that this site makes it apparent that shark attacks are extremely rare occurrences, while providing an easily accessible resource for those wishing to know more about the subject.
+
+# Reference
```

Comment:
`References` and `Publisher` should be combined into `Sources` and the data become like this (Check the Raw)

[Shark Research Institute](http://www.sharkattackfile.net/index.htm)